paimon-web-server/src/main/java/org/apache/paimon/web/server/service/impl/MetadataServiceImpl.java

```diff
@@ -157,11 +173,32 @@ public List<DataFileVO> getDataFile(MetadataDTO dto) {
         try {
             reader.forEachRemaining(
                     internalRow -> {
-                        DataFileVO dataFileVo = new DataFileVO();
-                        dataFileVo.setPartition(internalRow.getString(0).toString());
-                        dataFileVo.setBucket(internalRow.getInt(1));
-                        dataFileVo.setFilePath(internalRow.getString(2).toString());
-                        dataFileVo.setFileFormat(internalRow.getString(3).toString());
+                        DataFileVO dataFileVo =
+                                DataFileVO.builder()
+                                        .partition(internalRow.getString(0).toString())
+                                        .bucket(internalRow.getInt(1))
+                                        .filePath(internalRow.getString(2).toString())
+                                        .fileFormat(internalRow.getString(3).toString())
+                                        .schemaId(internalRow.getLong(4))
+                                        .level(internalRow.getInt(5))
+                                        .recordCount(internalRow.getLong(6))
+                                        .fileSizeInBytes(internalRow.getLong(7))
+                                        .minKey(
+                                                internalRow.getString(8) != null
+                                                        ? internalRow.getString(8).toString()
+                                                        : "")
```

Comment:
Ditto.

Reply:
During the test, it was found that internalRow.getString(8) may return null, and if no judgment is made, a null pointer will be reported.